lighthouse-core/gather/gatherers/dobetterweb/tags-blocking-first-paint.js

```diff
@@ -104,6 +86,7 @@ function filteredAndIndexedByUrl(networkRecords) {
     // Include 404 scripts/links generated by the parser because they are likely blocking.
     if (isHtml || isParserScriptOrStyle || (isFailedRequest && isParserGenerated)) {
       prev[record._url] = {
+        isLinkPreload: record.isLinkPreload,
```

Comment:
IMO it's not the end of the world to have it fire false positives for a while since there's now a mitigation that won't fire if it didn't actually occur before the paint